<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SouvenirAndTrip</class>
 <widget class="QDialog" name="SouvenirAndTrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1555</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="TripAndSouvenirTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1561</width>
     <height>771</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>View One Team's Souvenirs</string>
    </attribute>
    <widget class="QPushButton" name="Home_PushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>670</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Souvenir_Select_Team_ComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>661</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select the Team you want to See Souvenirs for</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="Souvenir_TableWidget">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>110</y>
       <width>1071</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Green Bay Trip</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>881</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Green Bay Trip - From One stadium to another</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Green_Bay_Select_Stadium_ComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select Your Destination Stadium</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Green_Bay_Confirmation_PushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="greenBay_tableWidget">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>90</y>
       <width>471</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Stadium Name</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="greenBay_cart_button">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>350</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Go to Cart</string>
     </property>
    </widget>
    <widget class="QLabel" name="greenBay_distance_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>600</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Total_Distance_Label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>520</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>520</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Distance: </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Custom Trip</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>631</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Custom Trip- Go wherever from wherever</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Custom_Trip_Select_Stadium_ComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select Your Starting Stadium</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>140</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select cities you want to go to</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Confirm_Custom_Trip_PushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>440</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm Trip</string>
     </property>
    </widget>
    <widget class="QListWidget" name="Select_Cities_ListWidget">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>180</y>
       <width>741</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="customTrip_cart_button">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>440</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Go to Cart</string>
     </property>
    </widget>
    <widget class="QLabel" name="customTrip_distance_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>590</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>New England Trip</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>1091</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Visit all other teams from New England Stadium. All cities are prechosen so that you travel the shortest distance!</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Confirm_New_England_Trip_PushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm Trip</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Starting Stadium:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New England!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newEngland_cart_button">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>540</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Go to Cart</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="newEngland_tableWidget">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>100</y>
       <width>451</width>
       <height>501</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="newEngland_distance_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>670</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>Shortest Trip from Custom Start</string>
    </attribute>
    <widget class="QComboBox" name="Shortest_Distance_Select_Stadium_ComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select Your Starting Stadium</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>881</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Visit all other teams from wherever you choose. The remaining cities are prechosen so that you travel the shortest distance!</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Confirm_Custom_Shortest_Trip_PushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm Trip</string>
     </property>
    </widget>
    <widget class="QPushButton" name="shortestCustomTrip_cart_button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>460</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Go to Cart</string>
     </property>
    </widget>
    <widget class="QLabel" name="shortestCustomTrip_distance_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>630</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableWidget" name="shortestCustomTrip_tableWidget">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>170</y>
       <width>441</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_15">
    <attribute name="title">
     <string>Minimum Spanning Tree</string>
    </attribute>
    <widget class="QPushButton" name="Confirm_MST_Trip">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>410</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm Trip</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>861</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Travel from a prechosen destination using the MST. You will travel the shortest distance possible.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="mst_tableWidget">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>601</width>
       <height>531</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="mst_distance_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>660</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_16">
    <attribute name="title">
     <string>Minnesota Trip</string>
    </attribute>
    <widget class="QPushButton" name="Confirm_Minnesota_Trip_PushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>410</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm Trip</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>771</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Visit all other teams from Minnesota Vikings. The remaining cities are prechosen using the depth first algorithm!</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Starting Stadium:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Minnesota Vikings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minnesota_cart_button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>410</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Go to Cart</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="minnesota_tableWidget">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>130</y>
       <width>461</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="minnesota_distance_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>670</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_17">
    <attribute name="title">
     <string>Los Angeles Trip</string>
    </attribute>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Starting Stadium:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>771</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Visit all other teams from Los Angeles Rams. The remaining cities are prechosen using the breadth first algorithm!</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Confirm_Lo_sAngeles_Rams_Trip_PushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>410</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Confirm Trip</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Los Angeles Rams</string>
     </property>
    </widget>
    <widget class="QPushButton" name="losAngeles_cart_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>410</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Go to Cart</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="losAngeles_tableWidget">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>120</y>
       <width>451</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="losAngeles_distance_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>590</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
